spec/lib/psulib_traject/workers/hourly_indexer_spec.rb

```diff
@@ -1,14 +1,42 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require 'sidekiq/testing'
 
-RSpec.describe PsulibTraject::Workers::HourlyIndexer, skip: ci_build? do
-  let(:indexer) { described_class.new }
+Sidekiq::Testing.fake!
+
+RSpec.describe PsulibTraject::Workers::HourlyIndexer do
+  let(:indexer) { described_class }
+
+  before(:all) do
+    redis = Redis.new
+    redis.keys('hr:*').map { |key| redis.del(key) }
+  end
+
+  before do
+    ConfigSettings.symphony_data_path = './spec/fixtures'
+  end
 
   describe '#perform' do
     it 'submits jobs for each hourly file' do
-      # TBD
-      indexer.perform
+      indexer.perform_async
+      expect(described_class).to have_enqueued_sidekiq_job
+    end
+
+    it 'increases the size of the job queue' do
+      expect do
```

Comment:
We use { } on the do/end blocks here. I think it's because this is a lambda here? But I'm not sure that's 100% accurate.

Reply:
This, I'm not sure. it looks super weird to me, but was yanked from the rspec examles. the `{}` syntax works fine, I think I got it right.